src/statistics/backend_stats_context.cpp

```diff
@@ -24,13 +24,19 @@
 namespace peloton {
 namespace stats {
 
-BackendStatsContext& BackendStatsContext::GetInstance() {
+CuckooMap<std::thread::id, std::shared_ptr<BackendStatsContext>>
+    BackendStatsContext::stats_context_map_;
```

Comment:
It was declared inside the class, here I am define it. For const static variable, you can both declare and define inside the class, but here it's not a constant.